### PR TITLE
Fixes include/excluded update in the panel

### PR DESCRIPTION
**Pull request by NejcZdovc**
Resolves https://github.com/brave/brave-browser/issues/3480
Resolves https://github.com/brave/brave-browser/issues/3476

## Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/brave-browser/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [x] Added/updated tests for this change (for new code or code which already has tests).
- Verified that these changes build without errors on
  - [ ] Windows
  - [x] macOS
  - [ ] Linux
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests`) on
  - [ ] Windows
  - [x] macOS
  - [ ] Linux
- [ ] Verified that all lint errors/warnings are resolved (`npm run lint`)
- [ ] Ran `git rebase master` (if needed).
- [ ] Ran `git rebase -i` to squash commits (if needed).
- [x] Tagged reviewers and labelled the pull request as needed.
- [ ] Request a security/privacy review as needed.
- [x] Add appropriate QA labels (QA/Yes or QA/No) to include the closed issue in milestone

## Test Plan:
- enable rewards
- add some sites to ac table
- make sure that exclude/restore works correctly in settings page
- in new window to one page that is in your ac table
- open panel
- try toggling include and make sure that toggle updates and settings page updates
- toggle off current site in the panel
- (don't close panel) click on restore in settings page and make sure that panel reflects this change (toggle is set to include)

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Request a security/privacy review as needed.
- [ ] Adequate test coverage exists to prevent regressions 
- [x] Verify test plan is specified in PR before merging to source

### CODE REVIEW

File: components/brave_rewards/browser/rewards_service_impl.h [hunk 1]
```diff
@@ -10,6 +10,7 @@
 #include <map>
 #include <memory>
 #include <string>
+#include <utility>
```
Comment (jasonrsadler) [inline components/brave_rewards/browser/rewards_service_impl.h]: Is this for something in this PR or something else?

Comment (NejcZdovc) [reply components/brave_rewards/browser/rewards_service_impl.h]: this fixes linter error

**Comment by NejcZdovc** [comment]
@jasonrsadler so code didn't changed for adding stuff to ac table. Could you please test it on master and see if  you see the same problem?

**Comment by jasonrsadler** [comment]
Yes, that seems to be a separate issue outside this PR

**Comment by jasonrsadler** [comment]
@NejcZdovc can you rebase `brave-browser` and re-init. I'm unable to build this currently.

**Pull request merged**